packages/mobile/src/components/CurrencyDisplay.tsx

```diff
@@ -104,12 +105,14 @@ type FormatFunction = (amount: BigNumber.Value, currency?: CURRENCY_ENUM) => str
 function getFormatFunction(formatType: FormatType): FormatFunction {
   switch (formatType) {
     case FormatType.Default:
-      return getMoneyDisplayValue
+      return (amount: BigNumber.Value, _currency?: CURRENCY_ENUM) =>
+        getMoneyDisplayValue(amount, _currency)
     case FormatType.CentAware:
       return (amount: BigNumber.Value, _currency?: CURRENCY_ENUM) =>
         getCentAwareMoneyDisplay(amount)
     case FormatType.Fee:
-      return (amount: BigNumber.Value, _currency?: CURRENCY_ENUM) => getFeeDisplayValue(amount)
+      return (amount: BigNumber.Value, _currency?: CURRENCY_ENUM) =>
+        getFeeDisplayValue(amount, false, _currency === CURRENCY_ENUM.GOLD)
```

Comment:
is the currency always CELO for this case? seems like there might be cases where that's not true

Reply:
Yes, this is a style issue: the least invasive way to specify the correct number of decimals for each use-case was through the displayDecimals field of the CURRENCIES enum. It may be worth creating a separate switch case for fiat currencies where we want four decimals to be displayed (e.g., in the detailed dropdown of the Fee Drawer, as here), but the requested behavior (at least at the time of this work) would be identical to the requested default behavior for CELO.

Reply:
I meant yes as in "yes you raise a good point" not "yes this will always be CELO." It is not guaranteed that FormatType.Fee will be used for only CELO-denominated fees.